From a8d6f9ab1fe50435d9287317e82a2b48e7fe9415 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 16 Mar 2018 18:50:16 +0000
Subject: [PATCH] Add bin subfolder

In order for binary targets to build bin must exist, so add it to the
repo and remove it from the package's .gitignore.
---
 .gitignore     | 1 -
 bin/.gitignore | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)
 create mode 100644 bin/.gitignore

diff --git a/.gitignore b/.gitignore
index 8217e6b..053bd40 100644
--- a/.gitignore
+++ b/.gitignore
@@ -7,7 +7,6 @@
 *.o
 *.orig
 /pause/pause.o
-/bin/
 /test/bin2img/bin2img
 /test/checkseccomp/checkseccomp
 /test/copyimg/copyimg
diff --git a/bin/.gitignore b/bin/.gitignore
new file mode 100644
index 0000000..f935021
--- /dev/null
+++ b/bin/.gitignore
@@ -0,0 +1 @@
+!.gitignore
-- 
2.16.2

